R/Learner.R

```diff
@@ -388,6 +428,12 @@ Learner = R6Class("Learner",
       self$state$model
     },
 
+    #' @field bundled (`logical(1)` or `NULL`)\cr
+    #' Indicates whether the model has been bundled (`TRUE`), unbudled (`FALSE`), or neither (`NULL`).
```

Comment:
Is there a more elegant solution? Somehow strange that you always have to use `isTRUE()` because of the three states.